inbm-lib/inbm_lib/constants.py

```diff
@@ -71,3 +71,11 @@
 
 # Device local cache
 CACHE = str(INTEL_MANAGEABILITY_CACHE_PATH_PREFIX / 'repository-tool/')
+
+# OTA log file location
+LOG_FILE = get_canonical_representation_of_path("/var/cache/manageability/update-status.log")
```

Comment:
needs path that works on Windows and Linux

Reply:
probably str(INTEL_MANAGEABILITY_CACHE_PATH_PREFIX / 'update-status.log') would be fine (double check though)